### ISSUE TITLE
Skirt armature issues when exporting with KKBP_Exporter compared to KKMeshExporter

### ISSUE THREAD

**Issue by ilcane87** [FlailingFog/KK-Blender-Shader-Pack#62]
Hello, I've been following the development of this project for a while, so let me start by thanking all contributors for the incredible work you've been doing.

As for the topic at hand, a while ago I had successfully imported a character with version 4.3.1 of the addon (using KKMeshExporter), and got an accurate skirt armature for it:

![image](https://user-images.githubusercontent.com/34897042/140782880-7007869a-1e8a-494c-b12a-908836b66233.png)

Today I tried importing the same character with version 5 (using KKBP_Exporter), and this is the resulting skirt armature:

![image](https://user-images.githubusercontent.com/34897042/140783181-bc135e56-8d45-4dd4-a12c-13e4d7ecb11d.png)

The second one produces a different skirt shape from Koikatsu, which causes heavy clipping that needs manual fixing.

Here is the character I've been using, for reference:

https://db.bepis.io/koikatsu/view/17005

**Comment by MediaMoots** [comment]
Thanks! I'll try to fix this.

**Comment by ilcane87** [comment]
I forgot to mention that I used the default outfit for the character I exported.

By the way, the armature being the problem is just my assumption, it may be a mesh issue that is reflected on the armature shape.
If I had to take a wild guess, I'd try disabling the automatic setting of the T-Pose when pressing the export button in Koikatsu, since that pays a brief animation (even if you're already in T-Pose) that may interfere with the current state of the mesh during the export process.
